frontend/src/store.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
```

Comment:
Was it intentional to include this file in this PR? Because I think it's a different task.

Reply:
Also, the inclusion of the store is present in #25 , so I think we can keep this PR strictly on the config of the dev server proxy.

Reply:
Yup, I've reverted my PR to just the dev server proxy! Contacted jane after I checked the Jira board and realised she was working on that.